packages/orbit-components/src/Tabs/components/Tab/index.tsx

```diff
@@ -123,25 +25,68 @@ const Tab = ({
   }, [onChange, selected, setSelected]);
 
   return (
-    <StyledTab
+    <button
       data-test={dataTest}
+      className={cx(
+        "flex flex-1 flex-row items-center justify-between",
+        "appearance-none flex-row border-0",
+        "min-w-fit",
+        "relative box-border",
+        "font-base font-medium",
+        "rounded-t-normal",
+        "duration-fast transition-[background] ease-in-out",
+        "px-md",
+        compact ? "text-normal py-[5px] leading-normal" : "text-large leading-large py-[9px]",
+        disabled
+          ? "cursor-not-allowed opacity-50"
+          : [
+              "cursor-pointer",
+              !isSelected && "border-0",
+              isSelected && type === TYPE_OPTIONS.DEFAULT && "border-product-normal border-b-2",
+              isSelected &&
+                type !== TYPE_OPTIONS.DEFAULT && [
+                  type === TYPE_OPTIONS.BASIC &&
+                    "border-b-2 [border-image-slice:1] [border-image-source:theme(backgroundImage.tab-bundle-basic-foreground)]",
+                  type === TYPE_OPTIONS.MEDIUM &&
+                    "border-b-2 [border-image-slice:1] [border-image-source:theme(backgroundImage.tab-bundle-medium-foreground)]",
+                  type === TYPE_OPTIONS.TOP &&
+                    "border-b-2 [border-image-slice:1] [border-image-source:theme(backgroundImage.tab-bundle-top-foreground)]",
+                ],
+            ],
+        type === TYPE_OPTIONS.DEFAULT && "bg-white-normal hover:bg-white-normal-hover ",
+        type === TYPE_OPTIONS.BASIC &&
+          "bg-tab-bundle-basic-background hover:bg-tab-bundle-basic-background-hover active:bg-tab-bundle-basic-background-active",
+        type === TYPE_OPTIONS.MEDIUM &&
+          "bg-tab-bundle-medium-background hover:bg-tab-bundle-medium-background-hover active:bg-tab-bundle-medium-background-active",
+        type === TYPE_OPTIONS.TOP &&
+          "bg-tab-bundle-top-background hover:bg-tab-bundle-top-background-hover",
+      )}
       onClick={ev => {
         if (onClick) {
           onClick(ev);
         } else setSelected(index);
       }}
       type="button"
       disabled={disabled}
-      $type={type}
       role="tab"
       aria-selected={isSelected}
       aria-disabled={disabled}
       aria-controls={`panel-${index}`}
-      compact={compact}
-      active={isSelected}
     >
-      <StyledTabText type={type}>{children}</StyledTabText>
-    </StyledTab>
+      <span
+        className={cx(
+          "text-ink-dark w-full",
+          type !== TYPE_OPTIONS.DEFAULT && [
+            "bg-clip-text [-webkit-background-clip:text] [-webkit-text-fill-color:transparent]",
+            type === TYPE_OPTIONS.BASIC && "bg-tab-bundle-basic-foreground",
```

Comment:
I guess it's the matter of personal perference. I found it easier to read when it's in array, rather than having explicit longer condition for each of them, but I will apply as u suggesting, no strong opinion on this :)